.github/workflows/docs.yml

```diff
@@ -69,4 +73,10 @@ jobs:
         run: |
           cmake -B clang-build -GNinja -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS="clang" -DLLVM_ENABLE_SPHINX=ON ./llvm
           TZ=UTC ninja -C clang-build docs-clang-html docs-clang-man
+      - name: Build libc docs
+        if: steps.docs-changed-subprojects.outputs.libc_any_changed == 'true'
+        run: |
+          cmake -B libc-build -GNinja -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_RUNTIMES="libc" -DLLVM_ENABLE_SPHINX=ON ./runtimes
```

Comment:
this also needs `-DLIBC_INCLUDE_DOCS=ON`

Reply:
This doesn't seem necessary to set explicitly? It gets its initial value from `LLVM_INCLUDE_DOCS` which is set to `ON` by default.

Happy to add the flag if so desired though.

Reply:
ah, you're right. No need to add the extra flag.